ouroboros-network/test/Test/Mux.hs

```diff
@@ -148,6 +156,51 @@ newtype DummyPayload = DummyPayload {
 instance Show DummyPayload where
     show d = printf "DummyPayload %d\n" (BL.length $ unDummyPayload d)
 
+-- |
+-- Generate a byte string of a given size.
+--
+genByteString :: Int64 -> Gen BL.ByteString
+genByteString size = MkGen $ \(QCGen g0) _ ->
```

Comment:
Using `MkGen` is a bit low level. You can use `chooseAny` to get a seed, and then `SM.mkSMGen`:
```
g0 <- SM.mkSMGen =<< chooseAny
```

Reply:
nice, I'll fix this.